src/main/java/backend/controller/UserController.kt

```diff
@@ -282,10 +282,11 @@ class UserController(private val userService: UserService,
         sponsor.address = sponsorView.address?.toAddress() ?: sponsor.address
         sponsor.isHidden = sponsorView.isHidden ?: sponsor.isHidden
         sponsor.company = sponsorView.company ?: sponsor.company
-        sponsor.logo = sponsorView.logo?.let(::Media) ?: sponsor.logo
+        sponsor.logo = sponsorView.logo?.let(::Media) ?: null

```

Comment:
This change seems pretty dangerous. If someone forgets to include the logo it gets deleted

Reply:
Also you don't need the Elvis operator in that case